pandas/core/arrays/base.py

```diff
@@ -1494,6 +1496,48 @@ def _empty(cls, shape: Shape, dtype: ExtensionDtype):
             )
         return result
 
+    def _quantile(
+        self: ExtensionArrayT, qs: npt.NDArray[np.float64], interpolation: str
+    ) -> ExtensionArrayT:
+        """
+        ExtensionArray compatibility layer for quantile_with_mask.
+
+        We pretend that an ExtensionArray with shape (N,) is actually (1, N,)
+        for compatibility with non-EA code.
```

Comment:
This docstring would need to be updated to be written for EA implementors (eg for those it doesn't matter this is a compatility layer for quantile_with_mask)

Reply:
Good point, will update